iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/normalize/AffMatchAffiliationNormalizer.java

```diff
@@ -0,0 +1,68 @@
+package eu.dnetlib.iis.wf.affmatching.normalize;
+
+import java.io.Serializable;
+
+import com.google.common.base.Preconditions;
+
+import eu.dnetlib.iis.common.string.LenientComparisonStringNormalizer;
+import eu.dnetlib.iis.common.string.StringNormalizer;
+import eu.dnetlib.iis.wf.affmatching.model.AffMatchAffiliation;
+
+/**
+ * 
+ * Normalizer of {@link AffMatchAffiliation} properties
+ * 
+ * @author Łukasz Dumiszewski
+*/
+
+public class AffMatchAffiliationNormalizer implements Serializable {
+
+    
+    private static final long serialVersionUID = 1L;
+
+    
+    private StringNormalizer organizationNameNormalizer = new LenientComparisonStringNormalizer();
+    
+    private StringNormalizer countryNameNormalizer = new LenientComparisonStringNormalizer();
+    
+    private StringNormalizer countryCodeNormalizer = new LenientComparisonStringNormalizer();
+    
+    
+    
+    //------------------------ LOGIC --------------------------
+    
+    /**
+     * Normalizes properties of <code>affMatchAffiliation</code> with corresponding normalizers (see
+     * setters)<br/><br/>
+     * This method <b>changes the passed object and returns it</b>. It does NOT create a new one.
```

Comment:
Well, in this case it was safe. It was supposed to work faster than the version that creates new objects. 
However, your doubt has convinced me that the solution with creating new objects will be cleaner so I've changed the code appropriately (we can always return to the original solution if the problems with efficiency really emerge).